packages/viz/package.json

```diff
@@ -65,7 +65,9 @@
     "@loaders.gl/mvt": "^2.1.2",
     "@luma.gl/core": "8.1.0",
     "@mapbox/vector-tile": "^1.3.1",
+    "@types/chroma-js": "^2.0.0",
     "cartocolor": "^4.0.2",
+    "chroma-js": "^2.1.0",
```

Comment:
chroma is really nice. If we eventually use d3 for formatting, we could change to d3-scale, which has a pretty similar API:

```js
var color = d3.scaleLinear()
    .domain([10, 100])
    .range(["brown", "steelblue"]);
color(20); // "#9a3439"
color(50); // "#7b5167"
```